app/css/header.css

```diff
@@ -0,0 +1,41 @@
+/* .header {
+  height: 500px;
+} */
+
+.arduino-logo {
+    width: 460%;
+    margin-top: -305%;
+    grid-column: 2/3;
+}
+  
+.date {
+    width: 88%;
+    grid-column: 7/8;
+    margin-top: -240%;
+    margin-left: 20%;
+}
+  
+.time {
+    width: 180%;
+    grid-column: 9/10;
+    margin-top: -240%;
+    margin-left: -25%;
+}
+  
+.david {
+    width: 120%;
+    grid-column: 10/-2;
+    margin-top: 20%;
+    margin-left: -10%;
+    position: relative;
```

Comment:
don't see top, bottom, left,rigth property to move the element why you put relative position ?